### PR TITLE
Update index.zh-CN.md

### PR DESCRIPTION
**Pull request by Richard-Choooou**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |          更新表格 expandable onExpand配置文档|

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 858ffa0</samp>

Fixed the parameter order of the `onExpand` function in the table component. Updated the documentation in `components/table/index.zh-CN.md` to reflect the change.

### <samp>🤖 Generated by Copilot at 858ffa0</samp>

* Swap the order of the `onExpand` function parameters to match other event handlers ([link](https://github.com/ant-design/ant-design/pull/44811/files?diff=unified&w=0#diff-1d555de1f30d5e46ff95e81b3d10b27baf15321e455557e7d3a55ca09ff61209L247-R247))

### CODE REVIEW

File: components/table/index.zh-CN.md [hunk 1]
```diff
@@ -244,7 +244,7 @@ const columns = [
 | indentSize | 展示树形数据时，每层缩进的宽度，以 px 为单位 | number | 15 |  |
 | rowExpandable | 设置是否允许行展开 | (record) => boolean | - |  |
 | showExpandColumn | 设置是否展示行展开列 | boolean | true | 4.18.0 |
-| onExpand | 点击展开图标时触发 | function(record, event) | - |  |
+| onExpand | 点击展开图标时触发 | function(event, record) | - |  |
```
Comment (kiner-tang) [inline components/table/index.zh-CN.md:247]: 为什么改这个？
image
我看代码里类型定义和调用都是先传 record 的

Comment (afc163) [reply components/table/index.zh-CN.md:247]: +1

Comment (afc163) [reply components/table/index.zh-CN.md:247]: 看上去不应该改这个。

Comment (Richard-Choooou) [reply components/table/index.zh-CN.md:247]: image
image

但是实际对外的接口是，事件在先，record 在后？

Comment (Richard-Choooou) [reply components/table/index.zh-CN.md:247]: version: 5.5.0

Comment (Richard-Choooou) [reply components/table/index.zh-CN.md:247]: @afc163 @kiner-tang

Comment (Richard-Choooou) [reply components/table/index.zh-CN.md:247]: image

Comment (Richard-Choooou) [reply components/table/index.zh-CN.md:247]: 看了一下，貌似是 expandIcon 配置的 onExpand 事件，record 在先，event 在后。
这个是新的 expandable 的配置里的 onExpand，参数与上述相反，文档描述应该是有误的。

**Pull request closed without merging**